tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -291,3 +299,21 @@ Verify CPU And Memory Requests And Limits Are Defined For All Containers In All
     FOR    ${pod_info}    IN    @{project_pods_info}
         Verify CPU And Memory Requests And Limits Are Defined For Pod    ${pod_info}
     END
+
+Wait Until Operator Reverts "Grafana" To "Prometheus" In Rhods-Monitor-Federation
+    [Documentation]     wait and checks Operator have changed app grafana to prometheus
+    Sleep    10m    msg=wait to operator reverts the Changes
+    Wait Until Keyword Succeeds    5min    30s    Verify In Rhods-Monitor-Federation App Is    expected_app_name=prometheus
+
+Verify In Rhods-Monitor-Federation App Is
+    [Documentation]     Verifies in rhods-monitor-federation, app is showing ${expected_app_name}
+    [Arguments]         ${expected_app_name}
+    ${data} =    OpenShiftCLI.Get    kind=ServiceMonitor   namespace=redhat-ods-monitoring    field_selector=metadata.name==rhods-monitor-federation
+    ${app_name}    Set Variable    ${data[0]['spec']['selector']['matchLabels']['app']}
+    Should Be Equal    ${expected_app_name}    ${app_name}
+
+Replace "Prometheus" With "Grafana" In Rhods-Monirot-Federation
+    [Documentation]     Replace app to "Prometheus" with "Grafana" in Rhods-Monirot-Federation
+    OpenShiftCLI.Patch    kind=ServiceMonitor
```

Comment:
Can you use OpenShiftLibrary instead of  OpenShiftCLI?

Reply:
Thanks @jgarciao